.circleci/config.yml

```diff
@@ -76,6 +78,31 @@ jobs:
           name: Lint
           command: golangci-lint run
 
+  pa11y:
+    docker:
+      - image: circleci/node
+    resource_class: small
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 18.06.0-ce
+          docker_layer_caching: false
+      - run:
+          name: Retrieve pact
+          command: |
+            curl -u "$PACT_BROKER_USERNAME:$PACT_BROKER_PASSWORD" "$PACT_BROKER_URL/pacts/provider/sirius/consumer/sirius-user-management/version/$(echo $CIRCLE_SHA1 | cut -c -7)" -o sirius-user-management-sirius.json
+            docker volume create pacts_data
+            docker container create --name temp -v pacts_data:/root hello-world
+            docker cp ./sirius-user-management-sirius.json temp:/root/sirius-user-management-sirius.json
```

Comment:
You can't bind mount a directory in Circle on the docker executor, so decided to go down this route